### PR TITLE
Update local block adapter and local cache dir to go under lakefs_data

### PR DESCRIPTION
**Pull request by nopcoder**

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -29,15 +29,15 @@ import (
 
 const (
 	DefaultBlockStoreType                    = "local"
-	DefaultBlockStoreLocalPath               = "~/lakefs/data"
+	DefaultBlockStoreLocalPath               = "~/lakefs_data/data"
```
Comment (arielshaqed) [inline config/config.go:32]:
```suggestion
	DefaultBlockStoreLocalPath               = "~/data/lakefs/store"
```
or
```suggestion
	DefaultBlockStoreLocalPath               = "~/data/lakefs/block"
```

File: config/config.go [hunk 2]
```diff
@@ -29,15 +29,15 @@ import (
...
 	DefaultBlockStoreS3Region                = "us-east-1"
 	DefaultBlockStoreS3StreamingChunkSize    = 2 << 19         // 1MiB by default per chunk
 	DefaultBlockStoreS3StreamingChunkTimeout = time.Second * 1 // or 1 seconds, whatever comes first
 
 	DefaultCommittedLocalCacheRangePercent          = 0.9
 	DefaultCommittedLocalCacheMetaRangePercent      = 0.1
 	DefaultCommittedLocalCacheBytes                 = 1 * 1024 * 1024 * 1024
-	DefaultCommittedLocalCacheDir                   = "~/lakefs/local_tier"
+	DefaultCommittedLocalCacheDir                   = "~/lakefs_data/cache"
```
Comment (arielshaqed) [inline config/config.go:40]:
```suggestion
	DefaultCommittedLocalCacheDir                   = "~/data/lakefs/cache"
```

**Review by arielshaqed — APPROVED** [review]
I would still prefer to avoid the word "lakefs" in the top-level.  This is a *personal preference*, as discussed vf2vf, and my suggestions are just that.

Thanks!

**Review by arielshaqed — APPROVED** [review]
Excellent change! :grin: 

Should we also change `Dockerfile` to build these directories in the docker image?

**Comment by nopcoder** [comment]
> Excellent change! 😁
> 
> Should we also change `Dockerfile` to build these directories in the docker image?

:) Next PR will update the local adapter

**Pull request merged**